CONTRIBUTING.md

```diff
@@ -98,15 +101,30 @@ $ git commit
 Writing good commit logs is important. A commit log should describe what
 changed and why. Follow these guidelines when writing one:
 
-1. The first line should be 50 characters or less and contain a short
-   description of the change. All words in the description should be in
-   lowercase with the exception of proper nouns, acronyms, and the ones that
-   refer to code, like function/variable names. The description should
-   be prefixed with the name of the changed subsystem and start with an
+#### Commit Message Guidelines
+
+1. The first line (the header) should:
+   - contain a short description of the change.
+   - be 50 characters or less.
+   - be entirely in lowercase with the exception of proper nouns, acronyms, and the ones that
+   refer to code, like function/variable names.
+   - be prefixed with the name of the changed subsystem and start with an
    imperative verb. Example: "net: add localAddress and localPort
-   to Socket"
+   to Socket", "src: fix typos in node_lttng_provider.h"
+   - be meaningful; it is what other people see when they
+   run `git shortlog` or `git log --oneline`.<br>
+   Check the output of `git log --oneline files_that_you_changed` to find out
+   what subsystem (or subsystems) your changes touch.
 2. Keep the second line blank.
-3. Wrap all other lines at 72 columns.
+3. Wrap all other lines in the body at 72 columns.
```

Comment:
I added it for that same reason as https://github.com/nodejs/node/pull/11732#discussion_r104805066, where it's later referred to as the `body`.